payments/mobile-pay/after-payment.md

```diff
@@ -214,5 +214,5 @@ documentation_section="mobile-pay" %}
 [payee-reference]: /payments/mobile-pay/other-features#payee-reference
 [abort]: /payments/mobile-pay/after-payment#abort
 [technical-reference-callback]: /payments/mobile-pay/other-features#callback
-[technical-reference-payeeReference]: /payments/mobile-pay/other-features#payee-reference
+[payee-reference]: /payments/mobile-pay/other-features#payee-reference
```

Comment:
Codacy found an issue: [[no-duplicate-definitions] Do not use definitions with the same identifier (214:1)](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5810356)